server/schedulers/hot_region.go

```diff
@@ -893,6 +911,13 @@ func (bs *balanceSolver) betterThan(old *solution) bool {
 	return false
 }
 
+func (bs *balanceSolver) getRkCmpPriorities(old *solution) (firstCmp int, secondCmp int) {
+	fk, sk := getRegionStatKind(bs.rwTy, bs.firstPriority), getRegionStatKind(bs.rwTy, bs.secondPriority)
+	fkRkCmp := rankCmp(bs.cur.srcPeerStat.GetLoad(fk), old.srcPeerStat.GetLoad(fk), stepRank(0, 100))
+	skRkCmp := rankCmp(bs.cur.srcPeerStat.GetLoad(sk), old.srcPeerStat.GetLoad(sk), stepRank(0, 10))
```

Comment:
In general, we use 100 as the step rank of byte dim and 10 as the step rank of key dim.